packages/utilities/gel-foundations/CHANGELOG.md

```diff
@@ -1,13 +1,13 @@
 # Gel Foundations Changelog
 
-| Version | Description |
-|---------|-------------|
-| 0.2.4   | [PR#323](https://github.com/BBC/psammead/pull/323) Update readme storybook badge |
-| 0.2.3   | [PR#264](https://github.com/BBC/psammead/pull/319) Resolving package-lock issues. |
-| 0.2.2   | [PR#304](https://github.com/BBC-News/psammead/pull/304) Add font family definitions, which include GEL-defined font fallbacks. |
-| 0.2.1   | [PR#303](https://github.com/BBC-News/psammead/pull/303) Bump to ensure correct build is published |
-| 0.2.0   | [PR#290](https://github.com/BBC-News/psammead/pull/290) Move typography font-sizes to use rem instead of em |
-| 0.1.4   | [PR#224](https://github.com/BBC-News/psammead/pull/224) Add tests for the exported values, coverage 100% :tada: |
-| 0.1.1   | [PR#212](https://github.com/BBC-News/psammead/pull/212) Update package description and README. |
+| Version | Description                                                                                                                                                  |
+| ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ |
```

Comment:
Looks like this file got caught by an auto-linter, can you reset the formatting please? 🙈